pkg/controller/acmechallenges/sync.go

```diff
@@ -39,6 +39,10 @@ import (
 
 const (
 	reasonDomainVerified = "DomainVerified"
+	CleanUpError         = "CleanUpError"
+	PresentError         = "PresentError"
+	Presented            = "Presented"
+	Failed               = "Failed"
```

Comment:
It's long-winded, but the convention is to give constants a common prefix which, see `reasonDomainVerified` .

Reply:
And all these constants should be private until someone comes along and asks for them to be public, at which point we'd probably consider adding them to the apis package. e.g. `reasonCleanupError` rather than `CleanupError`